iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergExpireSnapshots.java

```diff
@@ -42,4 +43,26 @@ public void testExpireSnapshotsWithTimestamp() throws IOException, InterruptedEx
     table.refresh();
     Assert.assertEquals(2, table.history().size());
   }
+
+  @Test
+  public void testExpireSnapshotsWithSnapshotId() throws IOException, InterruptedException {
+    TableIdentifier identifier = TableIdentifier.of("default", "source");
+    Table table = testTables.createTableWithVersions(shell, identifier.name(),
+        HiveIcebergStorageHandlerTestUtils.CUSTOMER_SCHEMA, fileFormat,
+        HiveIcebergStorageHandlerTestUtils.CUSTOMER_RECORDS, 10);
+    Assert.assertEquals(10, IterableUtils.size(table.snapshots()));
+
+    // Expire one snapshot
+    shell.executeStatement(
+        "ALTER TABLE " + identifier.name() + " EXECUTE EXPIRE_SNAPSHOTS" +
```

Comment:
weird syntax, why not
````
ALTER TABLE <table_name> EXPIRE SNAPSHOTS [older_than <value>] [ids]
````

Reply:
we already have a syntax in place, discussed offline, we can keep it same for now & explore an alt in another ticket